utils/auto_dockerfile_update/get_dockerfiles.py

```diff
@@ -49,26 +49,43 @@ def parse_base_image(full_base_image_name: str) -> (str, str, str):
     return repository, image_name, tag
 
 
-def is_dev_only(dockerfile_path=str) -> bool:
+def read_build_conf(dockerfile_path: str) -> str:
```

Comment:
lets have this return a key-value pair dictionary instead, with keys and values both stripped of surrounding spaces. This way this can be used in the future and we wont have an issue about issues with spaces in the file